### ISSUE TITLE
Not finding movies by 'search'

### ISSUE THREAD

**Issue by tymanthius** [galli-leo/Radarr#27]
Just an fyi, as this isn't fully under your control.

I have torznab indexers set up via jackett for BeyondHD, PTP, TorrentLeech (for searching) and the built in TorrentLeech for RSS feeds.

When I search for movies that I know are on both TL & PTP, no results.

Also, do you have a forum for more general discussion anywhere?

Provide a description of the feature request or bug, the more details the better.
Please use https://forums.sonarr.tv/ for support or other questions. (When in doubt, use the forums)

**Comment by fedoranimus** [comment]
We do not have forums at this time, but we do have a [Discord server](https://discord.gg/AD3UP37) for discussion.

**Comment by galli-leo** [comment]
I think the problem is that Jackett does not expose search functionality for movies with Torznab. I will have to implement the TorrentPotato tracker, so that searching works better.

**Comment by tymanthius** [comment]
Thanks for the update.  :)

**Comment by galli-leo** [comment]
@tymanthius I have implemented the TorrentPotato provider. If you want to test it, you can download the latest release here: http://leonardogalli.ch/radarr/builds/2017/01/05/Radarr_Windows_16%3A17%3A11.zip

**Comment by tymanthius** [comment]
Will that run under mono?  I need a user tag that says 'mono version'  ;)

**Comment by galli-leo** [comment]
@tymanthius This should do the trick: http://leonardogalli.ch/radarr/builds/2017/01/05/Radarr_Mono_16%3A17%3A11.zip

**Comment by tymanthius** [comment]
I'm getting 403 forbidden errors on some TorrentPotato items (BeyondHD, TorrentLeech Search), and it just refuses to admit that the PTP one is there.

If I disable the TP items other than PTP, it claims there are no searchable indexers.

If I do tests with TP settings, they fail, but if I test with Torznab, tests succeed. 

Side note: I had to manually change the categories from 5000 stuff to 2000.

**Comment by galli-leo** [comment]
Can you post the logs of the errors?

**Comment by tymanthius** [comment]
Here's a snippet, let me know if you need more:

https://hastebin.com/ebatiyihon.cs

**Comment by galli-leo** [comment]
Ah, you have to create a new Indexer and selecting TorrentPotato. Just changing the url in the Torznab indexer won't work.

**Comment by tymanthius** [comment]
Doh . . . .

**Comment by tymanthius** [comment]
That's mostly done the trick. Still doesn't seem to search PTP via jackett tho. :/

But it does search at least TorrentLeech via Jackett.  

Here's a pastebin of logs w/ notes on what I tried:

https://hastebin.com/ucetabeliz.xml

Note: Jacket seems to work just fine, and in CP I don't put a user name at all for TP stuff.

**Comment by galli-leo** [comment]
Are you sure you configured the PTP tracker correctly in Jackett? To me it seems to be an issue with Jackett.

**Comment by galli-leo** [comment]
Jackett just ignores the username, so you can put anything there. However, Radarr needs a username, so you should be able to put anything in there.

**Comment by tymanthius** [comment]
PTP is (at least now) set up correctly.  It passes the test.   But still won't return results.  

Not a huge deal, this is just more of an FYI.  :)

**Comment by galli-leo** [comment]
Can you share the log again?

**Comment by tymanthius** [comment]
Here's the log from right after start up, then hitting search on Ghostbusters: https://hastebin.com/xafafotoku.xml

**Comment by galli-leo** [comment]
Can you post the log of Jackett?

**Comment by tymanthius** [comment]
Jackett log: https://hastebin.com/anuwiyuret.js

**Comment by galli-leo** [comment]
Can you try to readd the PTP indexer with localhost instead of the ip adress?
Also, can you try just PTP (delete the others)?

**Comment by tymanthius** [comment]
After doing that, this is what radarr log shows:

17-1-5 13:54:27.2|Info|NzbSearchService|Searching 1 indexers for [Ghostbusters]
17-1-5 13:54:28.3|Info|DownloadDecisionMaker|Processing 22 releases


However, I don't see anything in the gui.

**Comment by galli-leo** [comment]
Did you use manual or automatic search?

**Comment by tymanthius** [comment]
I click the search tab, and it starts searching.  As I don't get results, I don't get an option for manual or automatic.

However, when I was getting results, I could click 'close' and then have the man/auto buttons.

**Comment by galli-leo** [comment]
Does the log say anything after Processing 22 releases?

**Comment by tymanthius** [comment]
Radarr logs show rocessing 22 releases, but status shows 'no idexers enabled' 

It looks like the legacy sonarr code doesn't recognize PTP's TP at all.

**Comment by galli-leo** [comment]
Can you post the full log after processing 22 releases?

**Comment by tymanthius** [comment]
That was it.

**Comment by galli-leo** [comment]
This was with only PTP enabled?

**Comment by tymanthius** [comment]
Correct.

**Comment by fedoranimus** [comment]
It would be great to move these discussions to the [support channel on Discord](https://discord.gg/AD3UP37) as to not pollute people who are following this thread.

**Comment by fedoranimus** [comment]
Has this issue been resolved?

**Comment by schumi2004** [comment]
Better to comment on this issue instead of opening a new one. I got the same behaviour and can reproduce it.
I searched for the following 3 movies.
query: 
xXx to find xXx: Return of Xander Cage
Split
Kedi

It returned no results and log showed:
Error occured while searching for new movies.: String was not recognized as a valid DateTime.

Then i searched for the same 3 titles but now including year, got instant results.
query:
xXx: Return of Xander Cage (2017)
Split (2016)
Kedi (2016)

Here are the log files:
https://www.dropbox.com/sh/x25g47xwyzbezx2/AAA1Az-CmhjOLiJKCtO_2ktsa?dl=0
Note: search is in reverse order after i found out it worked including year
